app/presenters/vacancy_presenter.rb

```diff
@@ -115,31 +115,32 @@ def greyed_additional_job_role(role)
     tag.span additional_job_role(role), class: "govuk-hint govuk-!-margin-bottom-0"
   end
 
-  # rubocop:disable Style/AsciiComments
   def fix_bullet_points(text)
-    # This is a band-aid solution for the problem where (particularly) job adverts contain `•` but
-    # do not contain corresponding newlines, resulting in inline bullets.
+    # This is a band-aid solution for the problem where (particularly) job adverts contain bullet point characters
+    # (not list elements), but do not contain corresponding newlines, resulting in inline bullets.
     bullet = "•"
     text = normalize_bullets(text, bullet)
     return text unless text&.count(bullet)&.positive?
 
-    paragraphs = text.split("\n") # Presume newline signals end of unordered list
-    paragraphs.map { |para|
-      if para.count(bullet).zero?
-        para
-      else
-        first_bulleted_line_idx = para.gsub(/\s+/, "").first == bullet ? 0 : 1
-        para.split(bullet).reject(&:blank?).map.with_index { |line, index|
-          item = "<li>#{line}</li>"
-          index == first_bulleted_line_idx ? "<ul>#{item}" : item
-        }.join.concat("</ul>")
-      end
-    }.join("\n")
+    text.split("<br>").map { |para|
+      next para if para.count(bullet) <= 1 # If paragraph only has one bullet point it is probably correctly formatted
+
+      first_bulleted_line_idx = strip(para).first == bullet ? 0 : 1
+      para.split(bullet).reject { |line| strip(line).blank? }.map.with_index { |line, index|
+        item = "<li>#{line.gsub(%r{(&nbsp;|<div>|</div>|<!--block-->)+}, '')}</li>"
```

Comment:
I feel that regex can probably be extracted out into a constant - it's used both here and in `#strip`